### DISCUSSION TITLE
Is it possible to get PPC data from Amazon SP-API?

### DISCUSSION THREAD

**Discussion by ykuni-software-developer** [amzn/selling-partner-api-models#3191] [General]
Hello, we want to get PPC data such as Clicks / Conversion Rates using Amazon SP-API. Is it possible? or do we need to use the Amazon Advertising API for this?

**Comment by ianrussel** [comment]
No.You need separate Amazon Advertising API.In my case, I `combined` Advertising and Sellingpartner api's in my project
